sdk/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items-mlflow.ipynb

```diff
@@ -991,9 +1293,9 @@
    "name": "python3-azureml"
   },
   "kernelspec": {
-   "display_name": "Python 3.8 - AzureML",
+   "display_name": "Python [conda env:rc_133] *",
```

Comment:
Minor: I think, your local set up value got picked up here. display_name, name and version. You can use previous.